packages/handleRichTextBoxKeyEventNavigation/handleRichTextBoxKeyEventNavigation.js

```diff
@@ -44,3 +73,67 @@ export default function(e, record) {
     }
     return false;
 }
+
+// Depends on the implementation, there might be nested div elements
+// for a record's rich text box DOMNode. However, record.focus()
+// will only focus on one of them. When tabbing to the previous focusable
+// element, we need to skip all the elements for record's DOMNode.
```

Comment:
we can likely distill this comment into "we need to skip all elements contained in the current record's DOM node to navigate to a focusable element outside the record"?

Reply:
ya this is better. haha I'm so bad at the wording.